{{ cookiecutter.repo_name }}/docs/contributor_guide/pre_commit_hooks.md

```diff
@@ -55,38 +57,43 @@ Next, audit the baseline that has been generated by running:
 detect-secrets audit .secrets.baseline
 ```
 
-When you run this command, you'll enter an interactive console and be presented with a list of high-entropy string
-and/or anything which _could_ be a secret, and asked to verify whether this is the case. By doing this, the hook will
-be in a position to know if you're later committing any _new_ secrets to the repository, and it will be able to alert
-you accordingly.
+When you run this command, you'll enter an interactive console and be presented with a
+list of high-entropy string and/or anything which _could_ be a secret, and asked to
```

Comment:
strings

Reply:
Done in commit [`178de00`](https://github.com/ukgovdatascience/govcookiecutter/pull/31/commits/178de0063a87f15b35fed7fbb8052cb81d12dbf9).